gatsby-config.js

```diff
@@ -11,7 +11,6 @@ module.exports = {
       slack: "https://slack.cncf.io",
       twitter: "https://twitter.com/opengitops",
       reddit: "https://www.reddit.com/r/GitOps",
-      facebook: "https://facebook.com/open-gitops",
```

Comment:
@scottrigby Facebook says:
> "the page may have been removed."

Any jokes? 😆
> "I had a GitOps joke, which I accidentally deleted it. But I think it'll be back in a minute."
> by @chanwit https://twitter.com/chanwit/status/1287110405907550208

Reply:
I ❤️ this joke lol